From: Matthew Bekkema <mat8913ftw@gmail.com>
Date: Sun, 4 Jan 2015 20:49:22 +0100
Subject: useless-depends

Removes the useless dependancies This was reported by dpkg-shlibs, that warned
that our binary was marked as depending on various libs without any reason as
no symbol were used.

Forwarded: no
---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 47fa5e2..63b73a6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -143,11 +143,6 @@ if(WIN32)
 else()
 	# Unix probably
 	if(BUILD_CLIENT)
-		find_package(X11 REQUIRED)
-		find_package(OpenGL REQUIRED)
-		find_package(JPEG REQUIRED)
-		find_package(BZip2 REQUIRED)
-		find_package(PNG REQUIRED)
 		if(APPLE)
 			FIND_LIBRARY(CARBON_LIB Carbon)
 			FIND_LIBRARY(COCOA_LIB Cocoa)
@@ -169,7 +164,6 @@ else()
 	endif(APPLE)
 	#set(CLIENT_PLATFORM_LIBS -lXxf86vm)
 	# This way Xxf86vm is found on OpenBSD too
-	find_library(XXF86VM_LIBRARY Xxf86vm)
 	mark_as_advanced(XXF86VM_LIBRARY)
 	set(CLIENT_PLATFORM_LIBS ${CLIENT_PLATFORM_LIBS} ${XXF86VM_LIBRARY})
 endif()
